components/overflow-group/overflow-group.js

```diff
@@ -271,7 +280,7 @@ class OverflowGroup extends RtlMixin(LocalizeCoreElement(LitElement)) {
 	}
 
 	render() {
-		const overflowMenu = this._getOverflowMenu();
+		const overflowMenu = this.getOverflowMenu();
```

Comment:
`getOverflowMenu` is the other function that components using the mixin have to create